client/src/Try/Loader.purs

```diff
@@ -112,7 +112,7 @@ makeLoader rootPath = Loader (go Object.empty <<< parseDeps "<file>")
                   deps = { name: _, path: Nothing } <$> shim.deps
                 pure { name, path, deps, src }
               Nothing ->
-                pure { name, path, deps: [], src: ffiDep name }
+                throwError ("FFI dependency not provided: " <> name)
```

Comment:
Maybe:
```
Compilation succeeded, but the following FFI dependency is missing:
- url

We don't provide FFI shims for all libraries in the PureScript package sets; you can't execute Node libraries on Try PureScript.
```
(Adapted from https://github.com/purescript/trypurescript/issues/265#issuecomment-1024964567 🙂 )

Reply:
Perhaps: "We don't provide FFI shims for all libraries; for example, Node libraries are not supported on Try PureScript. If you would like to suggest a new FFI shim be supported, please open an issue."

Reply:
Updated the copy here: 
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/26548438/152662497-145bc48d-fce8-45f6-9007-87d5d51fdef1.png">

However I also realized that rendering this error message prevents the compiled code from displaying, since both want to set to the `compiled` state ...

Reply:
I think it's fine to surface this error in the iframe instead of only in the console. However, if we find a better place to put it in the future, then I'm happy to move it around!